.nuxtrc

```diff
@@ -1 +1,2 @@
 telemetry.enabled=false
+pages=true
```

Comment:
this is enabling `pages` also for the vitest runtime tests we have - maybe instead run `nuxt-content-twoslash` from within `test/fixtures/basic-types` or similar?

Reply:
I originally tried to use `test/fixtures/basic-types` but the problem is that the types are generated with different paths, that are relative to the project root (`../../../../node_modules/*`) - which would need a heavy rewrite to every dts file to get them working in twoslash, so I thought it might be better to use the types from root dir to also be more consistent with the final users' env.